modules/veteran/app/controllers/veteran/v0/representatives_controller.rb

```diff
@@ -17,6 +18,18 @@ def find_rep
                  status: :not_found
         end
       end
+
+      private
+
+      def check_required_fields
+        if params[:first_name].blank? || params[:last_name].blank?
+          render json: {
+            errors: [
+              details: 'First name and Last name are required to complete this request', status: 422
```

Comment:
do you both default to `details` for errors? (over title)

Reply:
It's technically detail (no s) and it's part of the JSON API spec error response https://jsonapi.org/format/#errors

Reply:
we fell into the habit of using `title` for general messages, and `details` when the message included unique info (to the request) (that's how we interpreted jsonapi)

Reply:
@bastosmichael this has been updated

Reply:
still looks like `details` to me

Reply:
do we never use `title` or `code`?

Reply:
if you go into files_changed it should show as detail:

Reply:
weird. that's where i thought i was looking 🤷‍♀